shell/platform/android/io/flutter/app/FlutterFragmentActivity.java

```diff
@@ -18,14 +18,14 @@
  * Base class for activities that use Flutter who also require the use of the
  * Android v4 Support library's {@link FragmentActivity}. Applications that
  * don't have this need will likely want to use {@link FlutterActivity} instead.
- * <p/>
- * <strong>Important!</strong> Flutter does not bundle the necessary Android
+ *
+ * <p><strong>Important!</strong> Flutter does not bundle the necessary Android
  * v4 Support library classes for this class to work at runtime. It is the
  * responsibility of the app developer using this class to ensure that they
  * link against the v4 support library .jar file when creating their app to
- * ensure that {@link FragmentActivity} is available at runtime.
+ * ensure that {@link FragmentActivity} is available at runtime.</p>
  *
- * @see https://developer.android.com/topic/libraries/support-library/setup.html
+ * @see "https://developer.android.com/topic/libraries/support-library/setup.html"
```

Comment:
http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDDIEDI seems to indicate that you need an HTML anchor element here. Otherwise, it won't render as a link.